tmt/options.py

```diff
@@ -50,6 +51,12 @@
     click.option(
         '-q', '--quiet', is_flag=True,
         help='Be quiet. Exit code is just enough for me.'),
+    click.option(
+        '--log-topic',
+        metavar=f'[{"|".join(topic.value for topic in tmt.log.Topic)}]',
+        multiple=True,
+        type=str,
+        help='If specified, --debug and --verbose would emit logs only for these topics.')
```

Comment:
Should it really be 'only these topics'? If log emits only those selected then other debugging / verbose information will be missing. 
How about either - 'enable additional topics which are disabled by default' OR
have special value for 'non-topic' meaning I can enable all messages I want on the command line.

Reply:
Right, that's a definitely wrong description.